### ISSUE TITLE
Relationship panel no longer appears in resource instance select widget

### ISSUE THREAD

**Issue by jacobtylerwalls** [archesproject/arches#10741]
> I think there are some other bugs related to this UI. When I tried testing as of the latest merge from `dev/7.6.x`, the first issue is that this check in the resource-instance widget template  `<!-- ko if: self.relationship() -->` was returning false (undefined) for no apparent reason. Both resource models had ontologies set up and are relatable to each other. (This persisted when I switched from the feature branch to `dev/7.6.x`) When I removed that check from the htm file, the valueid for the relationship type appeared instead of the preflabel.

<img width="1483" alt="Screenshot 2024-03-18 at 4 33 41 PM" src="https://github.com/archesproject/arches/assets/8366985/ed1bad94-78c1-4299-b6fc-e3ef973851cf">

_Originally posted by @whatisgalen in https://github.com/archesproject/arches/issues/10667#issuecomment-2005319288_

**Comment by jacobtylerwalls** [comment]
This may not be the right fix, but it gets the panel working again:

```diff
diff --git a/arches/app/media/js/viewmodels/resource-instance-select.js b/arches/app/media/js/viewmodels/resource-instance-select.js
index 0715422a7..2e3d27598 100644
--- a/arches/app/media/js/viewmodels/resource-instance-select.js
+++ b/arches/app/media/js/viewmodels/resource-instance-select.js
@@ -264,7 +264,7 @@ define([
                 inverseOntologyProperty = graph.config.inverseOntologyProperty;
 
                 if (self.node && (!ontologyProperty || !inverseOntologyProperty) ) {
-                    self.relationship(self.node.config.graphs()?.[0]?.useOntologyRelationship);
+                    self.relationship(!!self.node.ontologyclass());
                     var ontologyProperties = self.node.config.graphs().find(function(nodeConfigGraph) {
                         return nodeConfigGraph.graphid === graph.graphid;
                     });
```

**Comment by jacobtylerwalls** [comment]
Thanks for adding this fix. I'll open a new issue for the valueid appearing instead of the prefLabel.

**Comment by jacobtylerwalls** [comment]
Fixed in #10667
